docs/source/guides/retrieving_secrets_with_bob.rst

```diff
@@ -0,0 +1,51 @@
+==================
+Retrieving Secrets
+==================
+
+Bob.  Speeding down the highway on the edge of the desert in search of The Secret.  But let's get to the heart of this thing:
+
+24 hours earlier, Bob had been sitting in the Pogo Lounge of the Beverly Heights Hotel (of course), when a call came from Alice: Bob had been Granted access.  And Enrico had shared a Capsule with Bob.
+
+On the advice of his Teacher Node, Bob rented a fast car with no top, and set out to find a cohort of *five specific* Ursulas - none of whom had ever even met one another.  In truth, Bob really only needed to convene with *any three* of these Ursulas for his gonzo journey to be complete.
+
+He needed to bring this Capsule around to each Ursula, who, upon inspecting it (but not being able to peer inside), were prepared to give him a CFrag to plug into one of the `n` slots on the Capsule.
+
+As soon as Bob completed plugging any `m` of these CFrags in, the Capsule glowed its sunny glow to show that it was ready for Bob's key; the final piece of the process of retrieve().
+
+----
+
+We sometimes jokingly refer to Bob as "Bob the BUIDLer" - but there is at least a small nugget of truth in this designatation.  A critical design decision about user-facing functionality in any NuCypher-powered decentralized application is where, when, and how to use the secrets that you retrieve using the network.
+
+It's not an exaggeration to say that the parameters you pass to `retrieve(...)` are a representation of the critique your application is making in the world.
+
+
+Basic retrieval
+---------------
+
+Alice has created a Policy and Granted Bob access.
+Enrico has given Bob a Capsule.
+Now Bob wants the message.
+
+```
+retrieve()
```

Comment:
That's a very fun guide. Am i right that it's not intended to tell how would *exactly* Bob retrieve?

Reply:
(e.g. what arguments to use, a self-sufficient example, ...)

Reply:
Yeah, as we talked about in Osaka, a subsequent PR in this same epic will advance the docs.